custom_components/tahoma/config_flow.py

```diff
@@ -61,3 +70,37 @@ async def async_step_import(self, import_config: dict):
                 return await self.async_validate_input(import_config)
             except Exception as exception:  # pylint: disable=broad-except
                 _LOGGER.exception(exception)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle a option flow for TaHoma."""
+
+    def __init__(self, config_entry):
+        """Initialize options flow."""
+        self.config_entry = config_entry
+        self.options = dict(config_entry.options)
+
+        if self.options.get(CONF_UPDATE_INTERVAL) is None:
+            self.options[CONF_UPDATE_INTERVAL] = DEFAULT_UPDATE_INTERVAL
```

Comment:
Is it possible as using voluptuous you set a default value? https://github.com/iMicknl/ha-tahoma/pull/227/files#diff-7132e279b007f1584ef563c8b28cdd00R102

Reply:
Unfortunately not, I did that first. However every time the OptionsFlow dialog opens, it will show the default value instead of the set value..